lib/assets/javascripts/jquery.are-you-sure.js

```diff
@@ -0,0 +1,192 @@
+/*!
+ * jQuery Plugin: Are-You-Sure (Dirty Form Detection)
+ * https://github.com/codedance/jquery.AreYouSure/
```

Comment:
Please add `gem "rails-assets-jquery.are-you-sure", "1.9.0"` to the `Gemfile` instead.